README.md

```diff
@@ -1,8 +1,53 @@
-# React + Vite
+# NC News Frontend
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+[Deployed Version](https://nc-news-zh.netlify.app/)
```

Comment:
I would move this into the links section below with the label deployed front end